index.html

```diff
@@ -37,9 +37,9 @@
 
```

Comment:
- [ ] You have done a great job implementing accessibility for your webpage but I would suggest adding the `alt` attribute for the languages in the About section. I think it could be a good idea to say that the image is a "JavaScript logo" or "HTML logo" :wink:
- [ ] There are a few contrast issues that I would suggest fixing for better accessibility. I think you could try updating the color or adding `font-weight` to the text. Remember that the primary color according to the Figma design is `#6060ff` :wink: :raised_hands:
  ![image](https://user-images.githubusercontent.com/81522435/139946113-f3494864-33a1-4dde-aaf2-ede47b99e34f.png)


  Logo color:
  ![image](https://user-images.githubusercontent.com/81522435/139945736-029dafe2-e47c-4b0c-a585-42054c139da7.png)

  See project button:
  ![image](https://user-images.githubusercontent.com/81522435/139945835-d61ebfe2-1e4e-41a8-8857-62351f43062b.png)

  Get in touch button (In this particular situation you could use the alternate colors so the button doesn't lose with the background when hovering):
  ![image](https://user-images.githubusercontent.com/81522435/139945888-8192c122-e0d5-4354-9028-4788b39fe4b7.png)

Reply:
Hi @alonsofl 
I appreciate your reviews, I have taken action on your comments and made the necessary modifications to pass the test of accessibility.